media_driver/linux/common/os/i915/mos_bufmgr.c

```diff
@@ -3267,11 +3267,6 @@ do_exec2(struct mos_linux_bo *bo, int used, struct mos_linux_context *ctx,
         i915_execbuffer2_set_context_id(execbuf, ctx->ctx_id);
     execbuf.rsvd2 = 0;
 
-#ifdef ANDROID
-    i915_execbuffer2_set_tag(execbuf, tag);
```

Comment:
If we don't call i915_execbuffer2_set_tag() remove definition as well.

Reply:
This is not addressed. I understand that you remove this in the next patch along with the whole i915_ext.h, but commits should be self-contained. You remove _tag in this commit, thus you need to remove declaration in _this_ commit